src/components/welcome/MyAccounts/AccountItem.tsx

```diff
@@ -59,6 +59,8 @@ const AccountItem = ({ onLinkClick, safeItem, safeOverview }: AccountItemProps)
     ? extractCounterfactualSafeSetup(undeployedSafe, chain?.chainId)
     : undefined
 
+  const isReplayable = !safeItem.isWatchlist && (!undeployedSafe || !isPredictedSafeProps(undeployedSafe.props))
```

Comment:
I think we also need to cover for this case of "old" counterfactual safes inside the `NetworkSelector`. Right now we display the loading skeletons indefinitely.
<img width="310" alt="Screenshot 2024-09-18 at 14 02 36" src="https://github.com/user-attachments/assets/ccf9a037-1f6c-49b2-b760-20693ddaec31">